packages/phone-number-privacy/migrations/20200330212301_create-number-pairs-table.ts

```diff
@@ -0,0 +1,12 @@
+import * as Knex from 'knex'
+
+export async function up(knex: Knex): Promise<any> {
+  return knex.schema.createTable('number_pairs', (t) => {
```

Comment:
Can we define these strings ('number_pairs', 'accounts', 'user_phone_hash', 'contact_phone_hash', etc) in a constants file?

Reply:
can do!